pkgs/development/python-modules/inotify_simple/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "inotify_simple";
```

I don't like this being `inotify_simple`.

```suggestion
  pname = "inotify-simple";
```

---

I also prefer dashes, but I suggest we should keep the original package name, as it improves discoverability and searchability. (That's my strong preference after all my experiences with searching for nixpgs.)

---

I'd really like to not have any additional packages named like this.

And it's not completely a preference as we have a guideline of sorts for this.
> https://nixos.org/nixpkgs/manual/#contributing
> Attribute names in python-packages.nix should be normalized according to PEP 0503. This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz )


So I'd think anyone searching for a python package in nixpkgs should expect their names to be normalized in this manner.

---

Sorry, I missed that contrib guideline.
Fixed: https://github.com/NixOS/nixpkgs/pull/44086/commits/93d0511927c88fd4816832129532c3c4040959ce